cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/dns/HeadlessServiceDnsBootstrap.scala

```diff
@@ -1,31 +1,31 @@
 /*
  * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.cluster.bootstrap.dns
+package akka.management.cluster.bootstrap.dns
 
-import java.util.{ Date, Locale }
+import java.util.Date
 
 import akka.actor.Status.Failure
 import akka.actor.{ Actor, ActorLogging, ActorRef, ActorSystem, Address, DeadLetterSuppression, Props, Timers }
 import akka.annotation.InternalApi
 import akka.cluster.Cluster
-import akka.cluster.ClusterEvent.CurrentClusterState
-import akka.cluster.bootstrap.{ ClusterBootstrap, ClusterBootstrapSettings }
+import akka.discovery.{ ServiceDiscovery, SimpleServiceDiscovery }
+import akka.discovery.SimpleServiceDiscovery.ResolvedTarget
 import akka.http.scaladsl.model.Uri
-import akka.pattern.{ ask, pipe }
-import akka.discovery.ServiceDiscovery
-import akka.discovery.ServiceDiscovery.ResolvedTarget
+import akka.management.cluster.bootstrap.{ ClusterBootstrap, ClusterBootstrapSettings }
 import akka.util.PrettyDuration
+import akka.pattern.pipe
 
 import scala.collection.immutable
 import scala.concurrent.{ Await, Future }
 import scala.concurrent.duration._
+import scala.util.Try
 
 /** INTERNAL API */
 @InternalApi
 private[bootstrap] object HeadlessServiceDnsBootstrap {
 
-  def props(discovery: ServiceDiscovery, settings: ClusterBootstrapSettings): Props =
+  def props(discovery: SimpleServiceDiscovery, settings: ClusterBootstrapSettings): Props =
```

Comment:
So much simpler now!

Reply:
Yeah I'm open to a better name. The name `ServiceDiscovery` sadly is taken by the akka extension hmm

Reply:
I'm ok with it, just found it funny.

Reply:
I guess it's simple because just one method of string to resolved thingy... Open to better names though, I'm not happy with the Simple prefix hm